app/controllers/internal/articles_controller.rb

```diff
@@ -39,6 +39,7 @@ def update
     article.update!(article_params)
     Article.where.not(id: article.id).where(live_now: true).update_all(live_now: false) if article.live_now
     CacheBuster.bust("/live_articles")
+    Audit::Logger.log(:moderator, current_user, params.dup)
```

Comment:
What if we make all of these a `after_action` at the top of the controllers? This way they don't need to be sprinkled around the code and it's super clear to the reader which actions are audited

Reply:
Great idea! Done!